docs/source/operations/orchestrator/deployment.rst

```diff
@@ -0,0 +1,223 @@
+******************************
+How to deploy the Orchestrator
+******************************
+
+This guide shows you how to deploy the Orchestrator component of Substra.
+
+Deployment
+==========
+
+Prerequisites
+-------------
+
+In order to deploy the Orchestrator you will need a fully configured Kubernetes cluster.
+
+Preparing your Helm values
+--------------------------
+
+The orchestrator deployment is packaged using Helm.
+You can find the complete description of values that can be used to configure the chart on `Artifact Hub <https://artifacthub.io/packages/helm/substra/orchestrator>`_.
+
+#. Add the Helm repository for Substra:
+
+   .. code-block:: bash
+
+      helm repo add substra https://substra.github.io/charts/
+
+#. Create a Helm values file named ``orchestrator-values.yaml`` with the following content:
+
+   .. code-block:: yaml
+
+      ingress:
+        enabled: true
+        hostname: HOSTNAME
+
+   | Replace ``HOSTNAME`` with the hostname of your Orchestrator.
+   | This will setup the ingress in order to make your Orchestrator accessible at the defined hostname.
+
+#. Setup your :term:`Substra channels<Channel>`.
+   In the ``orchestrator-values.yaml`` file, add the following content:
+        
+   .. code-block:: yaml
+
+      channels:
+        - name: CHANNEL-NAME
+          organizations: [ORG-NAME, ...]
+
+   | Replace ``CHANNEL-NAME`` with the name you want to give to your channel.
+   | Replace ``ORG-NAME, ...`` with the names of the :term:`Organizations<Organization>` that you want to participate in this channel.
+   | If you want more than one channels you can add more items in the list.
+
+Here you have created the most basic configuration required to have a running orchestrator that can support a Substra network.
+If you want you can jump directly to the section deploy or you can follow along the next sections to enhance the security of your orchestrator.
+
+Setup TLS for the Orchestrator
+------------------------------
+
+If you want to you can enable TLS for your orchestrator.
+For this you will need to generate a few certificates, here we will generate them manually but you can also use automated tools for this task.
+If you want to use automated tools we provide a certificate resource for `cert-manager <https://cert-manager.io/>`_, check out the ``orchestrator.tls.createCertificates`` values.
```

Comment:
Same here, we should highly recommend to use cert-manager. certificates are created with 1y validity, we should probably have a warning that this is the lifetime of the network (unless certs are updated).